.env.development

```diff
@@ -1,8 +1,8 @@
 # These are the default development settings.
 # To make changes to your specific instance, please modify .env
 
-CANONICAL_HOST=localhost
-CANONICAL_PORT=3000
+# CANONICAL_HOST=localhost
+# CANONICAL_PORT=3000
```

Comment:
It's because we had to set up an ssh tunnel to test out the chargify stuff in development. I can remove that stuff